azure-function-v4.js

```diff
@@ -0,0 +1,21 @@
+module.exports = azureFunctionV4;
+
+/**
+ * @param {import('probot').Probot} probot
+ * @param {import('@azure/functions').HttpRequest} request
+ * @param {import('@azure/functions').InvocationContext} context
+ * @returns {Promise<import('@azure/functions').HttpResponseInit>}
+ */
+async function azureFunctionV4(probot, request, context) {
+  await probot.webhooks.verifyAndReceive({
+    id: request.headers.get("X-GitHub-Delivery"),
+    name: request.headers.get("X-GitHub-Event"),
+    signature: request.headers.get("X-Hub-Signature-256") || request.headers.get("X-Hub-Signature"),
```

Comment:
I don't think there is any GHES version left that sends out the `X-Hub-Signature` header, I think we can just leave it out.

Reply:
Removed.